src/store/wizardSlice.ts

```diff
@@ -336,6 +353,29 @@ export const selectFirstBootScript = (state: RootState) => {
   return state.wizard.firstBoot?.script;
 };
 
+export const selectUsers = (state: RootState) => {
+  return state.wizard.users;
+};
+
+export const selectUserName = (state: RootState) => {
+  return state.wizard.userInfo?.userName;
+};
+
+export const selectUserPassword = (state: RootState) => {
+  return state.wizard.userInfo?.password;
+};
+
+export const selectConfirmUserPassword = (state: RootState) => {
+  return state.wizard.userInfo?.confirmPassword;
+};
+export const selectUserSshKey = (state: RootState) => {
+  return state.wizard.userInfo?.sshKey;
+};
+
+export const selectUserAdministrator = (state: RootState) => {
+  return state.wizard.userInfo?.administrator;
+};
+
```

Comment:
We wouldn't need any of these apart `selectUsers` as we would only work with the entire Users object at once.

Reply:
I actually think that we need those fields to simplified the process